auto_verify.sh

```
@@ -32,8 +32,11 @@ run_verification() {
     log_file="${log_dir}/$(echo "$image_name" | tr ':/' '_')_logs.txt"
     # If the user is root
     script_path="/build_script.sh"
+    if [ "$patch_available" == "yes"]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)